taxi/models.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63)
+    country = models.CharField(max_length=63)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["name", ],
+                                    name="unique_manufacturer_name"
+                                    ),
+        ]
+        ordering = ("name", )
+
+    def __str__(self):
+        return f"{self.name} ({self.country})"

```

Comment:
You are overusing f-string here. According to the checklist, you should avoid using f-strings when not necessary. Consider using a regular string concatenation or format method.